lib/operations/cursor_ops.js

```diff
@@ -20,39 +20,30 @@ function count(cursor, applySkipLimit, opts, callback) {
     if (typeof cursor.cursorSkip() === 'number') opts.skip = cursor.cursorSkip();
```

Comment:
just a note while we're here: `applySkipLimit` is documented as defaulting to true, but we don't actually do that anywhere. Also, it begs the question why `applySkipLimit` isn't in `options`

Reply:
yeah let's get rid of it, it's not even an option in the top-level `count`, can you actually set this thing anywhere?